tests/test_inputs_handler_matsim_reader.py

```diff
@@ -338,3 +340,23 @@ def test_reading_pt2matsim_vehicles():
                 'egressTime': {'secondsPerPerson': '0.5'},
                 'doorOperation': {'mode': 'serial'},
                 'passengerCarEquivalents': {'pce': '2.8'}}})
+
+
+def test_reading_HK_network_with_elevation():
```

Comment:
The test name doesn't fully communicate what we're testing. How about `test_reads_elevation_values_into_network_when_present_in_file` or something like that? The `does_something_when_something` naming pattern.